lib/segment/src/data_types/vectors.rs

```diff
@@ -210,6 +238,7 @@ impl VectorStruct {
             VectorStruct::Multi(vectors) => vectors.values().all(|v| match v {
                 Vector::Dense(vector) => vector.is_empty(),
                 Vector::Sparse(vector) => vector.indices.is_empty(),
+                Vector::MultiDense(v) => v.iter().all(|v| v.is_empty()),
```

Comment:
the function `VectorStruct::is_empty` is completely unused since you moved it to `schema.rs`

I propose to remove it a different PR and tackle the `is_empty` semantic when adding the API type

Reply:
Okay, let's remove it